apps/store/themes/store/js/categorization.js

```diff
@@ -63,21 +67,37 @@ var categorization = function () {
                 queryObjArray.push(queryObj);
             }
 
-            for (var i in queryObjArray) {
-                $('.categorization-checkbox:checkbox').each(function () {
-                    var $this = $(this);
-                    if ($this.attr('name') == queryObjArray[i].queryKey) {
-                        for (var k in queryObjArray[i].queryValue) {
-                            if ($this.attr('value').toLowerCase()
-                                == queryObjArray[i].queryValue[k].trim().toLowerCase()) {
-                                $this.attr('checked', true);
-                                $("#" + $this.attr('name')).collapse('show');
-                                var icon = $("#" + $this.attr('name')).prev().find('.status').children();
-                            }
-                        }
+            $('.refine > .panel > div').each(function () {
+                var field = $(this).attr('id');
+                var queryObject = queryObjArray.filter(function (queryObj) {
+                    return queryObj.queryKey == field;
+                });
+
+                if (queryObject.length == 0) {
+                    return;
+                }
+
+                var queryValues = (queryObject.length == 1) ? queryObject[0].queryValue : null;
```

Comment:
@splinter Thanks for pointing out. Fixed in #841 (c7e7f2fba36a308ff02c339614352b54567c54fa)